src/server/procedures/myForest/stats.ts

```diff
@@ -67,13 +67,62 @@ export const stats = procedure
         JOIN profile p ON g.recipient_id = p.id
         WHERE p.guid = ${profileId}
       `;
+    // console.log('giftData:', giftData);
+
+    const giftMetadata = await prisma.$queryRaw<CountryProjectStatsResult[]>`
+			SELECT (metadata->>'$.project.id') AS projectId, (metadata->>'$.project.country') AS country
+			FROM gift g 
+			JOIN profile p ON g.recipient_id = p.id
+			WHERE p.guid = ${profileId} AND g.purpose IN ('trees', 'conservation')
```

Comment:
Is this enough to limit the results to valid entries, e.g. do we have to exclude columns with `AND g.deleted_at IS NULL` like for contributions?

Reply:
I've added that in, thanks for letting me know.